src/backend/core/tests/test_models_invitations.py

```diff
@@ -73,6 +73,51 @@ def test_models_invitations__is_expired(settings):
     assert expired_invitation.is_expired is True
 
 
+def test_models_invitation__new_user__convert_invitation_to_access():
```

Comment:
We should test :
- One invitation to xyw@email.com converted to a team access ✅
- Several invitations to xyw@email.com converted to several team accesses
- Expired invitations kept in the database with no team access created ✅

Moreover, we test in isolation valid and expired invitations, we have no test to check that only valid invitations will be converted when an email has several invitations with some expired.